server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+
+const app = express()
+
+app.get("/ping",(req , res)=>{
+  res.send("pong")
+})
+
+app.listen(3000,()=>{
```

Comment:
It's a good practice to make the port number configurable through environment variables. This makes your application more flexible in different deployment environments. [important]